src/modules/App/App.js

```diff
@@ -34,14 +36,20 @@ const App = () => {
       .catch(error => setError(error))
   }, [])
 
+  useEffect(() => {
```

Comment:
Nice use of useEffect() - Now we don't need to render all of the missions if user isn't signed in and stumble through all of that additional logic